project/plugins.sbt

```diff
@@ -0,0 +1,4 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.6.0-SNAPSHOT")
```

Comment:
I take it -SNAPSHOT is just for bootstrapping? Do we actually need this, or might we as well do classpath shadowing?

Reply:
Yes, it's only for bootstrapping. We might be able to do something with classpath shadowing, but it would be difficult to ensure that we are testing the right thing, IMO.